tests/unit/Core/Framework/DataAbstractionLayer/Dbal/SchemaBuilderTest.php

```diff
@@ -422,7 +422,7 @@ protected function defineFields(): FieldCollection
             (new IdField('id', 'id'))->addFlags(new PrimaryKey(), new Required()),
             new VersionField(),
             new ParentFkField(self::class),
-            (new ReferenceVersionField(self::class, 'parent_version_id'))->addFlags(new Required()),
+            (new ReferenceVersionField(self::class, 'parent_version_id', 'parentVersionId'))->addFlags(new Required()),
```

Comment:
Can you add an example with a completely different property name and assert on that also? so we know the new behaviour is actually working.